game_eggs/steamcmd_servers/ark_survival_ascended/egg-ark--survival-ascended.json

```diff
@@ -111,10 +111,10 @@
             "field_type": "text"
         },
         {
-            "name": "Query Port",
-            "description": "ARK query port used by steam server browser and ark client server browser.",
-            "env_variable": "QUERY_PORT",
-            "default_value": "27015",
+            "name": "Game Port",
```

Comment:
This is already set by the primary allocation This is not needed.

Reply:
The SERVER_NAME does the same indeed. I just thought to make it more clear.